user/tests/wiring/ble_central_peripheral/ble_central/central.cpp

```diff
@@ -445,11 +448,14 @@ static void pairingTestRoutine(bool request, BlePairingAlgorithm algorithm,
         assertTrue(waitFor([&]{ return !BLE.isPairing(peer); }, 20000));
         assertTrue(BLE.isPaired(peer));
         assertEqual(pairingStatus, (int)SYSTEM_ERROR_NONE);
+// FIXME: le_bond_get_sec_level() failed to retrieve the security level
+#if !HAL_PLATFORM_RTL872X
```

Comment:
This is the only issue that still needs to be resolved.

Reply:
FAE  just send mee a patch that has exposed an API to get the security level even if connnection is not bonded.

Reply:
nice